### PR TITLE
Added documentation for new calibration GUI 

### PR DESCRIPTION
**Pull request by maxzieba**
## Proposed changes

User Story: https://shadowrobot.atlassian.net/browse/SP-378
Added documentation on method level to new GUI for Polhemus glove calibration.

## Checklist

Before posting a PR ensure that from each of the below categories **AT LEAST ONE BOX HAS BEEN CHECKED**. If more than one category is applicable then more can be checked. Also ensure that the proposed changes have been filled out with relevant information for reviewers.

## Tests

- [ ] No tests required to be added. (For small changes that will be tested by CI/CD infrastructure).
- [ ] Added/Modified automated and PhantomHand CI tests (if a new class is added (Python or C++), the interface of that class must be unit tested).
- [ ] Manually tested in simulation (if simulation specific or no hardware required to test the functionality). 
- [ ] Manually tested on hardware (if hardware specific or related).

## Documentation

- [ ] No documentation required to be added.
- [ ] Added documentation (For any new feature, explain what it does and how to use it. Write the documentation in a relevant space, e.g. Github, Confluence, etc).
- [ ] Updated documentation (For changes to pre-existing features mentioned in the documentation).

### CODE REVIEW

File: src/polhemus_ros_driver/knuckle_position_calibration_action_server.py [hunk 1]
```diff
@@ -72,6 +89,11 @@ class DataMarker(Marker):
     _id = 0
 
     def __init__(self, frame_id, point, color, size=0.002):
+        """
```
Comment (jeferal) [inline src/polhemus_ros_driver/knuckle_position_calibration_action_server.py]: Is this comment refered to this constructor?

**Review by georgiablanco — APPROVED** [review]

**Review by jeferal — APPROVED** [review]

File: src/polhemus_ros_driver/knuckle_position_calibration_action_server.py [hunk 2]
```diff
@@ -19,17 +19,34 @@
...
+        @param out_min: Minimal value of mapped output
+        @param in_max: Maximal value of mapped output
+    """
     return (value - in_min) * (out_max - out_min) / (in_max - in_min) + out_min
 
 
 def calculate_distance(point1, point2):
+    """
+        Returns the distance between two points.
+        @param point1: Second point
```
Comment (beatrizleon) [inline src/polhemus_ros_driver/knuckle_position_calibration_action_server.py]: Should it be like this?
```suggestion
        @param point1: First point
```

Comment (beatrizleon) [reply src/polhemus_ros_driver/knuckle_position_calibration_action_server.py]: Also maybe worth saying the format you are expecting points to be given

File: src/polhemus_ros_driver/knuckle_position_calibration_action_server.py [hunk 3]
```diff
@@ -19,17 +19,34 @@
...
+    """
     point1 = [point1.x, point1.y, point1.z]
     point2 = [point2.x, point2.y, point2.z]
     return np.linalg.norm(np.array(point1)-np.array(point2))
 
 
 def sphere_fit(data):
+    """
+        Returns radius, center points and residuals of fitted sphere on input data.
+        @param data: Input data
```
Comment (beatrizleon) [inline src/polhemus_ros_driver/knuckle_position_calibration_action_server.py]: what should input data contain?

File: src/polhemus_ros_driver/knuckle_position_calibration_action_server.py [hunk 4]
```diff
@@ -170,6 +199,11 @@ def _create_marker(self, finger, color):
         return int_marker
 
     def _create_control(self, quaternion, name):
+        """
+            Creates as InteractiveMarkerControl to allow the user to drag&move the solution marker
+            @param quaternion: Quaternion defining the rotations
+            @param name: Name of the marker definiding allowed motion inf format 'rotate/move_axis'
```
Comment (beatrizleon) [inline src/polhemus_ros_driver/knuckle_position_calibration_action_server.py]: what does inf means?

**Review by beatrizleon — CHANGES_REQUESTED** [review]

**Review by beatrizleon — APPROVED** [review]

**Pull request merged**